design-library/src/components/BccGraphic/BccGraphic.spec.ts

```diff
@@ -0,0 +1,51 @@
+import { describe, it, expect } from "vitest";
+
+import { mount } from "@vue/test-utils";
+import BccGraphic from "./BccGraphic.vue";
+
+const bannerSrc = "https://event.bcc.no/wp-content/uploads/2023/10/Background-7.png";
+const logoSrc = "https://event.bcc.no/wp-content/uploads/2023/11/BUK4_Logo-Main.svg";
+
+describe("BccGraphic", () => {
+  it("renders graphic", async () => {
```

Comment:
I think some more valuable tests in this file could be to test
- whether an icon/subcomponent is rendered on a condition
- whether the correct bannerSrc/logoSrc are rendered

I don't think the snapshots are super valuable in general as it's easy to just regenerate them without looking at why they actually failed. A specific test for the bannerSrc will tell you what is wrong if someone edits the file down the line and messes up something.

Reply:
(same comment on all other test files)

Reply:
Makes sense, done